src/app/tx/[tx]/MessageDisplays.tsx

```diff
@@ -0,0 +1,157 @@
+import dynamic from 'next/dynamic';
+import { ExternalLink } from '@/components/ExternalLink';
```

Comment:
are we using @/ across the codebase instead of relative paths now

Reply:
Up to debate, I find it really convenient